v2/internal/testdata/trace.go

```diff
@@ -0,0 +1,154 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
it would be much more readable if we load the trace from json file.

Reply:
Not sure there's already a stable JSON representation for traces in OpenTelemetry. I'll take a look, but would do it as part of another commit, if possible. This specific package (internaldata) is mostly a copy of existing code from otelcol, used in the test suite for this exporter.

Reply:
I removed this, and will track the addition of this in another issue: https://github.com/jaegertracing/jaeger/issues/2884